### PR TITLE
setup.cfg

### PR DESCRIPTION
**Pull request by priyanshurohilla**
Addresses #832 
i have tried to resolve the issue please review it

### CODE REVIEW

**Comment by Juanlu001** [comment]
Hi @priyanshurohilla, thanks for your pull request! It looks to me though that you're using setuptools config (in setup.cfg) which is not used in poliastro, and therefore these changes would have no effect. Please take a look at how `flit` does this.

**Comment by priyanshurohilla** [comment]
@Juanlu001 i looked up how flit does that and i found that the flit was made so that u don't have to ship any package. see this
https://github.com/takluyver/flit/issues/115#issuecomment-310930758

 It automatically detects and ships all the packages. So maybe what am i thinking is that we need to change the location of the `setup.cfg` file which is currently in poliastro folder to the folder where `flit` is located. Therefore, it will import 'setup.cfg' . This we have to do as there is currently no operation to import specific file from another location in `flit`.

see this
https://github.com/takluyver/flit/issues/115#issuecomment-310931356

**Comment by jorgepiloto** [comment]
I think [this](https://flit.readthedocs.io/en/latest/pyproject_toml.html#sdist-section) the part in which we should include something like:

```bash
[tool.flit.sdist]
include = [
    ...
    "src/poliastro/atmosphere/data/coesa76_rho.dat",
    "setup.cfg"
]
```

Not sure though :thinking:

**Comment by priyanshurohilla** [comment]
@jorgepiloto Actually the `sdist` command itself produces a minimal default set into the source distribution:

- all Python source files ` py_modules` and `packages`
- all C source files in the `ext_modules` or `libraries` 
- all test scripts: `test/test*.py`
- README.txt, setup.py and setup.cfg 

so basically it automatically imports that also, but in this case the file is present in another folder so i think explicitly including it will solve the issue.

**Comment by Juanlu001** [comment]
I think this should suffice! @priyanshurohilla can you please rebase or squash these commits into only one and `git push --force` so we can simplify the history?

File: src/poliastro/constants/general.py [hunk 1]
```diff
@@ -489,7 +489,7 @@
 R_polar_jupiter = Constant(
     "R_polar_jupiter",
     "Jupiter polar radius",
-    6.6854,
+    6.6854e7,
```
Comment (Juanlu001) [inline src/poliastro/constants/general.py]: This is because the branch is not up to date with current `master`. But let's not be picky with that now.

**Pull request merged**